packages/snap/test/unit/rpc/keyPairTestConstants.ts

```diff
@@ -1,3 +1,3 @@
 export const testAppKey = "test-app-key-6upaCfHM9RKAKU62Dg6upaCfH7g6upaCfH7";
 export const testPublicKey = "0x3f953693f8dfe6d4f9be593036e55c95fdd23f24e5fd3027893cf35538e0e31d";
-export const testAddress = "5DW5CXHWbM13Az7aetLQVUEviNq8WeXFQanHNPVMmzyRYKvX";
\ No newline at end of file
+export const testAddress = "E1grqdPDi1xvdw2Rb9TPRbvry7NKKLRrxd2m3mKFoBvHZqj";
```

Comment:
Is there a reason for this specific address, I am just curious.

Reply:
I've fixed address encoding (we had wrong value) and api resetting, also keypair now always returns address encoded for current configuration in network